src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-node/psbt-unsigned-input-list/components/psbt-unsigned-input-item.tsx

```diff
@@ -12,33 +14,34 @@ import { PsbtUnsignedInputWithInscription } from './psbt-unsigned-input-with-ins
 interface PsbtUnsignedInputItemProps {
   addressNativeSegwit: string;
   addressTaproot: string;
-  utxo: TaprootUtxo & OrdApiInscriptionTxOutput;
+  utxo: BitcoinTransactionVectorOutput;
+  utxoWithInscription: (TaprootUtxo & OrdApiInscriptionTxOutput) | undefined;
 }
 export function PsbtUnsignedInputItem({
   addressNativeSegwit,
   addressTaproot,
   utxo,
+  utxoWithInscription,
 }: PsbtUnsignedInputItemProps) {
   const calculateBitcoinFiatValue = useCalculateBitcoinFiatValue();
 
   const isInputCurrentAddress =
-    utxo.address === addressNativeSegwit || utxo.address === addressTaproot;
+    utxo.scriptpubkey_address === addressNativeSegwit ||
+    utxo.scriptpubkey_address === addressTaproot;
   const inputValue = satToBtc(utxo.value).toString();
-  const fiatValue = i18nFormatCurrency(calculateBitcoinFiatValue(utxo.value));
-  const inscription = utxo.inscriptions;
-
-  if (!utxo.address) return null;
+  const fiatValue = i18nFormatCurrency(calculateBitcoinFiatValue(inputValue));
```

Comment:
Minor nit/style I reckon makes sense, when it comes to view-level formatting calls, like `truncateMiddle` or `i18nFormatCurrency` think it makes sense to put them in JSX where used.